docs/guide/tutorial-en-us.md

```diff
@@ -1,42 +1,345 @@
-Tutorial
---------
+Tutorial in English :🇺🇸:
+==============================
 
-## How to "Hello World" ##
+3BC is a low-level, easy-to-learn programming language that works in the form of a virtual machine similar to a 1960s computer, but with a very unusual architecture that has processors of only **3 bits**.
 
-These steps will guide you to run the hello world example and to have your first 3BC program running in your machine, let get's started.
+Despite having an aspect to be an esoteric language, it has a good capacity for generic use to solve computational problems, and certain advantages with the implementation in embedded and microcontrollers.
 
-After downloading the binary on the [releases](https://github.com/RodrigoDornelles/3bc-lang/releases) page, just decompress the file and utilize the command line to interpret the source code using the compiler according to your operating system: **windows**, **mac os**, **unix** and **linux**.
+It was introduced by a Brazilian at Christmas 2020 after completing a month of development, being a project for studies and also as a conceptual proof of concept about punched cards being readable and practical for both humans and machines.
 
-### example: `helloworld.3bc` ###
+How to make a "Hello world!"
+--------------------------
+
+These steps will guide you through running the hello world example and getting your first 3BC program running on your machine, so let's get started.
+
+After downloading the binary from _[donwload](https://3bc-lang.org/download)_, simply unzip the file and use the command line to run the source code using the interpreter according to your operating system: **windows**, **mac os**, **unix** or **linux**.
+
+
+ * **Examples:** `helloworld.3bc`
 
 ```RUBY
-mode nill 0d2
-strc nill 'h'
-strc nill 'e'
-strc nill 'l'
-strc nill 'l'
-strc nill 'o'
-strc nill ' '
-strc nill 'w'
-strc nill 'o'
-strc nill 'r'
-strc nill 'l'
-strc nill 'd'
-strc nill '!'
+MODE NILL 0d2
+STRC NILL 'o'
+STRC NILL 'l'
+STRC NILL 'a'
+STRC NILL ' '
+STRC NILL 'm'
+STRC NILL 'u'
+STRC NILL 'n'
+STRC NILL 'd'
+STRC NILL 'o'
+STRC NILL '!'
 ```
 
-### windows ###
+ * **windows**
+
+using 'terminal' or 'cmd' type:
 
 ```
 C:\Users\nicod\Downloads\3bc-windows-32> 3bc.exe helloworld.3bc
 hello world!
 C:\Users\nicod\Downloads\3bc-windows-32>
 ```
 
-### mac / unix / linux ###
+ * **mac** / **unix** / **linux**
+
+using the shell of your choice type:
 
 ```
 $ ./3bc helloworld.3bc
 hello world!
 $
-```
\ No newline at end of file
+```
+
+Structure
+---------
+
+A great point to facilitate the learning, is its very well organized syntax of the language. With a quick view separated in rows and columns, where these have a fixed size of parameters, even when not used.
+
+A program line is consequently an instruction, which has three mandatory parameters. They are separated in the respectively order of **register**, **address**, **constant**.
+
+![structure_en><](https://3bc-lang.org/images/structure-pt-br.png)
```

Comment:
change to `![structure_en ><](https://3bc-lang.org/images/structure-en-us.png)`

Reply:
upload img if possible!

Reply:
ok